packages/destination-actions/src/destinations/pipedrive/pipedriveApi/deals.ts

```diff
@@ -0,0 +1,29 @@
+import { RequestClient } from "@segment/actions-core/src/create-request-client"
```

Comment:
```suggestion
import type { RequestClient } from "@segment/actions-core/src/create-request-client"
```

Also, you could add this to the exports of `core/src/index.ts` because I don't know if we want to support import paths like this. That way you could do `import type { RequestClient } from '@segment/actions-core'`